fraud/requester.go

```diff
@@ -0,0 +1,39 @@
+package fraud
+
+import (
+	"context"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+
+	"github.com/celestiaorg/go-libp2p-messenger/serde"
+
+	pb "github.com/celestiaorg/celestia-node/fraud/pb"
+)
+
+func requestProofs(
+	ctx context.Context,
+	host host.Host,
+	pid peer.ID,
+	proofTypes []pb.ProofType,
+) ([]*pb.ProofResponse, error) {
+	msg := &pb.FraudMessageRequest{RequestedProofType: proofTypes}
```

Comment:
Why is this an array? If it is part of the design, why is it designed like this?

Reply:
disagree that this should be already implemented, but okay, you can resolve